src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -229,13 +304,13 @@
  * Note: a <a href="https://checkstyle.org/config_filters.html#SuppressionXpathSingleFilter">
  * suppression xpath single filter</a> is needed because
  * IDEA has no blank line between "javax" and "java".
- * ImportOrder has a limitation by design to enforce an empty line between groups ("java", "javax").
- * There is no flexibility to enforce empty lines between some groups and no empty lines between
- * other groups.
+ * ImportOrder has a limitation by design to enforce an empty line between groups
+ * ("java", "javax"). There is no flexibility to enforce empty lines between some
+ * groups and no empty lines between other groups.
  * </p>
  * <p>
- * Note: "separated" option is disabled because IDEA default has blank line between "java" and
- * static imports, and no blank line between "javax" and "java".
+ * Note: "separated" option is disabled because IDEA default has blank line
+ * between "java" and static imports, and no blank line between "javax" and "java".
```

Comment:
Revert

Reply:
done

Reply:
I revert again bcoz of this ERROR:  ....... ......  "Line should not be longer than 100 symbols [lineLength] "

Reply:
@aryaniiit002 from newest master:
```
➜  checkstyle git:(master) awk '{print length, $0}'  src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java |sort -nr|head -2 
110  * <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#STATIC_IMPORT">
100      * Specify list of <b>static</b> import groups (every group identified either by a common prefix

```
The only existing line in this file that is over 100 characters is line 138, and Checkstyle makes exceptions for links. Somehow extra characters must be getting added if this is failing. If you ever have any doubts about your changes, you can use [git diff](https://git-scm.com/docs/git-diff) to see the difference between your branch's code, and master.

Reply:
OKay

Reply:
idk how but i am still getting this error.